Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -24,6 +24,32 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
         return textField
     }()
     
+    lazy var countryCodeLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Country Code:"
+        return label
+    }()
+    
+    lazy var countryCodeTextField: UITextField = {
+        let textField = UITextField()
+        textField.placeholder = "1"
+        textField.backgroundColor = .systemBackground
```

Comment:
what do you think about changing the keyboard type to phonePad?

`textField.keyboardType = .phonePad
`

Reply:
i like that! addressed in 52d24ce4639d984f6cc0f46592a763791e71753e